core/storage/trie/codec.hpp

```diff
@@ -11,6 +11,16 @@
 #include "storage/trie/node.hpp"
 
 namespace kagome::storage::trie {
+  /**
+   * @brief Stores encoded children node for later retrieval
+   */
+  struct StoreChildren {
```

Comment:
Hm, as far as I understand it's meant to store trie nodes, so wouldn't NodeStorage be a better name?